docs/design/jet/022-mongodb-connector.md

```diff
@@ -138,7 +136,7 @@ CREATE MAPPING people (
 )
 TYPE MONGODB 
 OPTIONS (
```

Comment:
there it should use `DATA CONNECTION "mongodb-ref"` syntax (I cannot make a suggestion as GitHub sees it as deleted line)